### DISCUSSION TITLE
Accrescent Store?

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#7976] [Ideas]
Hey,

could you publish Tuta Mail in the Accrescent Store? I would love to see you there and download the App from there.

Maybe you ask why we want this or what Benefits it would have for us as Users.

The Benefits for me would be a few, like these:

I could get Tuta Mail from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

It would make it a lot easier for me and save me time, because Tuta Mail is one of the few Apps, I use in Obtainium to download. And if my last few Apps would come to Accrescent, I wouldn't need to use Obtainium. I just use it until my last few Apps on there come to Accrescent.

Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.

I really hope that you Guys could make that possible, because it would have great Benefits for us as Users. And think like that: If we support them by adding Apps, we help them to grow from Alpha to Stable.

[Accresent features](https://accrescent.app/features) I think matter for Tuta Mail.
 
Security: 
[Signed repository metadata](https://accrescent.app/features#signed-metadata) 
[App signing key pinning](https://accrescent.app/features#key-pinning) 
[Name verification](https://accrescent.app/features#name-icon-verification)
 
Privacy: 
[No remote APK signing](https://accrescent.app/features#no-remote-signing) 
[No accounts](https://accrescent.app/features#no-accounts)

**Comment by chaitanyarajasekhar** [comment]
I would also love tuta to be on accrescent store because of split apk support, developer signed keys, unattended updates (not rate limited by github when downloading from obtainium). accrescent app is on grapheneos app store. It is also shown as a way for some apps to be downloaded on privacyguides.org

**Reply by Hasshu** [reply]
> accrescent app is on grapheneos app store.

For context: https://grapheneos.social/@GrapheneOS/112821386750410102

(Reposting: replied in the wrong place initially.)

**Comment by ghost** [comment]
Hello, is this planned yet?

**Reply by charlag** [reply]
Hey, we just checked and it doesn't seem like there's an API for submitting updates which is a no-go for us at the moment. We would have to wait until there is an API we can use.

**Reply by ghost** [reply]
> Hey, we just checked and it doesn't seem like there's an API for submitting updates which is a no-go for us at the moment. We would have to wait until there is an API we can use.

Yeah i understand, Accrescent dev himself is very busy at the moment so he doesn't have time to moderate and accepting devs request. I would love to see both Tuta Mail and Calender on Accrescent as soon as it development is back on track ❤️

**Comment by tribals** [comment]
In spite of recent Google policy to block free distribution of applications trough Play Store, I would like for Tuta to be in Accrescent.
